workflows/python/sdk/order-processor/app.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import threading
+from time import sleep
+from dapr.ext.workflow import WorkflowRuntime, DaprWorkflowClient
+from workflow_activities import order_processing_workflow, notify_activity, process_payment_activity, verify_inventory_activity, update_inventory_activity, requst_approval_activity
+from dapr.clients import DaprClient
+from model import InventoryItem, OrderPayload
+from util import get_address
+
+store_name = "statestore-actors"
+
+default_item_name = "cars"
+
+class WorkflowConsoleApp:    
+    def main(self):
+        print("*** Welcome to the Dapr Workflow console app sample!")
+        print("*** Using this app, you can place orders that start workflows.")
+        print("*** Ensure that Dapr is running in a separate terminal window using the following command:")
+        print("dapr run --dapr-grpc-port 4001 --app-id order-processor")
```

Comment:
I don't understand why we say this.  We control all the variables with this quickstart console app.

Reply:
Canwe remove?

Reply:
We did this in the .NET QuickStart because we were creating an interactive console session, which doesn’t work very well when running through the Dapr CLI.

Reply:
Hi, while i love the interaction scenario, I feel that's going to make it really hard to test in CI/cD.  Do we have an approach for this?

Reply:
No, this is for interactive scenario. The quickstarts here seem to serve more a purpose of e2e test. And, I didn't have this clue before I started to work on this PR. Hence, few such things can be seen.
I will remove it from here.